dashboard/lib/services/afe_enrollment.rb

```diff
@@ -47,7 +45,6 @@ def self.submit(first_name:, last_name:, email:, nces_id:, street_1:, street_2:,
       'school-zip' => zip,
       'inspirational-marketing-kit' => booleanize(marketing_kit),
       'csta-plus' => booleanize(csta_plus),
-      'aws-educate' => booleanize(aws_educate),
```

Comment:
it looks like this data gets posted to pardot.com. are you able to verify that pardot can handle this field being removed? it might be necessary to sign in to pardot and see what we have configured there.

Reply:
Ah, I see. Yes, I wasn't able to get to the "Congratulations" that I screenshotted when I tried to "break" the form, but I agree that I didn't actually check what gets sent to Pardot. I'll look into this –– thanks!